draft-wood-tls-external-psk-importer.md

```diff
@@ -97,20 +100,29 @@ supported HashAlgorithm 'hash', the importer constructs an ImportedIdentity stru
    } ImportedIdentity;
 ~~~
 
+ImportedIdentity.label SHOULD be bound to the protocol for which the key is imported. For example,
```

Comment:
If this is meant to support multiple protocols, the `HashAlgorithm` bit is probably a bit overly-specific. Maybe my other protocol doesn't use TLS hash algorithms, or maybe it uses the secret in a totally different way altogether. (See also the "hardcoded" SHA-256 down in the TLS 1.2 section.)

A thought: we could allocate labels like "tls13,sha256" and "tls13,sha384". Or maybe explicitly have two levels of labels?

Reply:
Interesting idea! I'll leave this as a "TODO" in the draft as a potential alternative.